### DISCUSSION TITLE
Border between

### DISCUSSION THREAD

**Discussion by MerlinMason** [tailwindlabs/tailwindcss#2299] [Ideas]
I'm really enjoying using the space between (`space-x` / `space-y`) feature and would also find a similar border-between useful too.

Something like:

```
item1  |  item2  |  item3
```

Could be `border-x-1` 

or 

```
item1
—————
item2
—————
item3
```

Could be `border-y-1`

Would be really useful if this worked in conjunction with space between, this is a pretty common layout pattern and would make styling it very easy.

**Comment by adamwathan** [comment]
Already exists! :)

https://tailwindcss.com/docs/divide-width

**Reply by MerlinMason** [reply]
`<Facepalm />`

Amazing, thanks!
